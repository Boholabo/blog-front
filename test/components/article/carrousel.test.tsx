import React from 'react';
import { render } from '../../testUtils';
import Carrousel from '../../../components/article/carrousel';

describe('article/carrousel component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <Carrousel article={{ data: { data: [] } }} />,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
