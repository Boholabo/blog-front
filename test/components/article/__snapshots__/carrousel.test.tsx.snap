// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article/carrousel component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="carrousel"
  >
    <div
      class="swiper index"
      style="transition: opacity 300ms ease-in-out; opacity: 0;"
    >
      <div
        class="swiper-wrapper"
      />
    </div>
    <div
      class="swiper-pagination swiper-pagination-clickable swiper-pagination-bullets"
    />
  </div>
</DocumentFragment>
`;
